src/dispatch/batch_builder.rs

```diff
@@ -0,0 +1,150 @@
+use crate::{
+    dispatch::{Dispatcher, DispatcherBuilder},
+    world::ResourceId,
+    Accessor, AccessorCow, DynamicSystemData, RunningTime, System, SystemData, World,
+};
+
+/// The `BatchBuilder` is responsible for the creation of the Batch.
+///
+/// The `Batch` is a Systam that have the capabilities to contain another dispatcher
+/// Thanks to this characteristic is possible to customize its dispatching
+///
+/// The BatchBuilder accept the dispatcher builder as parameter, and the type of the
+/// System that will drive the execution of the internal dispatcher.
+///
+/// Note that depending on the dependencies of the SubSystems the Batch
```

Comment:
"SubSystems" implies some kind of type; also, "dependencies" should probably be clarified by using "resource dependencies" instead.

On dependencies, we should also document cross-dispatcher dependencies don't work.